### ISSUE TITLE
Could not parse schema: Unexpected characters ().

### ISSUE THREAD

**Issue by black-snow** [jormaechea/open-api-mocker#10]
```
[FATAL] Could not parse schema at /mnt/f/projects/x/src/main/resources/x_oas3.yaml: Unexpected characters ()
```

This is all I get. Sadly, I cannot provide the schema but other tools have no issues reading it. I can't find "unexpected characters" in the code - I'm curious where it comes from.

**Comment by jormaechea** [comment]
@black-snow it’s kinda hard to help you without being able to reproduce the issue on my end. But lets try something: convert you OpenAPI spec into json and try again. I have a feeling that it’s an issue with the yaml parser.

Let me know if that worked.

**Comment by black-snow** [comment]
I was able to nail it down to my response `example` for a 200 `application/json` response in my OAS `3.0.3` spec:

```yaml
              example:
                content:
                  - meta:
                      id: "xxxxxxxxxxxxxxx"
                      creationDate: "2020-09-09T15:57:28.197Z"
                      updateDate: "2020-09-16T22:38:35.589Z"
                      rubric: "BP-ZH"
                      subRubric: "BP-ZH01"
                      language: "de"
                      registrationOffice:
                        id: "yyyyyy"
                        displayName: "abc"
                        street: "str"
                        streetNumber: "3"
                        swissZipCode: "12345"
                        town: "ghfgh"
                        containsPostOfficeBox: false
                        postOfficeBox: null
                        publicationOriginator: null
                        publicationNumber: "BP-ZH01-0000015864"
                        publicationState: "PUBLISHED"
                        publicationDate: "2020-09-17T00:00:00.000Z"
                        expirationDate: "2021-09-17T00:00:00.000Z"
                        primaryTenantCode: "fgjg"
                        onBehalfOf: null
                        invoiceAddressId: null
                        legalRemedy: "xyz"
                        cantons": [
                            "ZH"
                        ]
                        secondaryTenants: null
                        repeatedPublications: null
                        customsStampImages: []
                        title:
                          de: "gfdfg"
                          en: "dgsd"
                          it: "dfg"
                          fr: "dfhh"
                        dossierReference: null
                        copyDeadline: null
                    links: []
                    attachments: []
                    content: null
                    commented: false
                total: 135
                pageRequest:
                    sortOrders: []
                    page: 0
                    size: 3
```

**Comment by jormaechea** [comment]
You have an indentation error at the end of it:
```yaml
                pageRequest:
                    sortOrders: [] # Two extra spaces here
                    page: 0
                    size: 3
```

**Comment by black-snow** [comment]
Thanks for the quick reply, but I don't see what you mean. 😅

**Comment by jormaechea** [comment]
Well, I found out why this fails. The issue is with the json-like array in multiple lines.

You can make this change and it will work right away:
```diff
-                        cantons": [ # Note: you have an extra quote there...
-                            "ZH"
-                        ]
+                        cantons:
+                            - ZH
```

I think this is just an issue with the YAML parser we are using. Let me try an alternative later today and get back to you.

**Comment by black-snow** [comment]
Good catch. Funnily this also works fine: `cantons: ["ZH"]`.

**Comment by jormaechea** [comment]
@black-snow Just released v1.4.1 that adds support for JSON-array notation.

I'm closing this issue. If you have any problems, let me know.

**Comment by black-snow** [comment]
Well that was an easy fix :) Thanks for looking into this.
It'd be nice if we added a regression test for this.
